state/indexer/block/kv/kv.go

```diff
@@ -49,7 +54,7 @@ func (idx *BlockerIndexer) Has(height int64) (bool, error) {
 // The following is indexed:
 //
 // primary key: encode(block.height | height) => encode(height)
-// FinalizeBlock events: encode(eventType.eventAttr|eventValue|height|finalize_block) => encode(height)
+// FinalizeBlock events: encode(eventType.eventAttr|eventValue|height|finalize_block|eventSeq) => encode(height)
```

Comment:
In line 74 below, we now do not distinguish the different types of events and store in our database always the same string `finalize_block`. This is taking up space and is completely useless. I wonder whether we should, at some point open a PR to alter the indexer to remove this? (It is trivial to remove but we would need to account that previous versions of the indexer stored this `typ` field so we'd need to parse it)